pom/firstTask.js

```diff
@@ -0,0 +1,126 @@
+const Page = require('./page');
+
+
+class Condition extends Page {
+    get GoogleSearchInput(){
```

Comment:
Only classes should start with Capitals. Please rename methods and fields. Please check whole PR.